Classes/Base.php

```diff
@@ -278,4 +278,23 @@ public static function logout()
 			$session->set( $key, null );
 		}
 	}
+
+	/**
+	 * Doing a complete ACPu wipe with the red system cache.
+	 *
+	 * @hook clearCachePostProc
+	 *
+	 * @param array $cacheType The caching type.
+	 *
+	 * @return void
+	 */
+	public static function clearAPCuCache( array $cacheType )
+	{
+		if ( $cacheType['cacheCmd'] === 'all'
+			&& static::getExtConfig( 'useAPC', '0' ) === '1'
```

Comment:
I would only use `static::getExtConfig( 'useAPC' )` because it's not strictly defined what variable type is returned

Reply:
You mean like this?
```php
(bool) \Aimeos\Aimeos\Base::getExtConfig( 'useAPC', false ) === true
```
I just noticed that this is used at several other places in the code.

Reply:
Either that or use `&& static::getExtConfig( 'useAPC' ) &&` only

Reply:
Done.